### ISSUE TITLE
Support local timezone

### ISSUE THREAD

**Issue by gazal-k** [jenkins-x/jx-pipelines-visualizer#42]
Tried setting `TZ` environment variable on jx-pipelines-visualizer `Deployment` which did not seem to make any difference. It might be even better if it rendered date/time based on locale of user.

**Comment by vbehar** [comment]
Yes, for the moment we only support UTC. We could have a timezone selector in the UI, to let the user pick the right one.

**Comment by gazal-k** [comment]
I was thinking more like, just returning an ISO8601 back from server side and let client side create a javascript `Date` object. I think that'll use the browser timezone.
